examples/nlp/duplex_text_normalization/data/data_split.py

```diff
@@ -51,6 +51,7 @@ def read_google_data(data_file: str, lang: str, split: str):
         data_file: Path to the data file. Should be of the form output-xxxxx-of-00100
         lang: Selected language.
         split: data split
+        add_test_full: do not truncate test data
```

Comment:
could you clarify this a bit: is this taking the whole test file not #num of lines?